wolfcrypt/src/asn.c

```diff
@@ -8282,7 +8412,32 @@ int ParseCertRelative(DecodedCert* cert, int type, int verify, void* cm)
             }
         }
     }
-
+#if defined(WOLFSSL_RENESAS_TSIP)
+    /* prepare for TSIP TLS cert verification API use */
+    if (cert->keyOID == RSAk) {
+        /* to call TSIP API, it needs keys position info in bytes */
+        RsaPublicKeyDecodeRawIndex(cert->publicKey, (word32*)&idx,
```

Comment:
Return code needs to be checked here for RsaPublicKeyDecodeRawIndex().

Reply:
Added to check the return code if RsaPublicKeyDecodeRawIndex() is failure.